### PR TITLE
First commit of Sprint 1

### PR DESCRIPTION
**Pull request by JonGeorge**
This PR is to merge change into the **phase-II branch**.

The changes included in the PR are as follows:
- Added the ability to get restaurant data for a given session id and return the data to the client
- Updated documentation to reflect dependencies (i.e. Python3.9) and the steps required for setting up the backend for local development

### CODE REVIEW

File: aws_serverless/template.yaml [hunk 1]
```diff
@@ -81,6 +81,37 @@ Resources:
...
+        - AWSSecretsManagerGetSecretValuePolicy:
+            SecretArn:
+              arn:aws:secretsmanager:us-east-1:600091550595:secret:yelp_key-M2asns
+        - DynamoDBWritePolicy:
+            TableName:
+              Grubfinder_Session
+        - DynamoDBWritePolicy:
+            TableName:
+              Grubfinder_User
+        - DynamoDBWritePolicy:
```
Comment (nabilamin) [inline aws_serverless/template.yaml:103]: Let's change this to "DynamoDBReadPolicy" for the Grubfinder_Restaurant policy and get rid of the others. Best to limit the scope of a lambda to limit attack surfaces.

File: aws_serverless/template.yaml [hunk 2]
```diff
@@ -81,6 +81,37 @@ Resources:
             Path: /api/session/create
             Method: post
 
+  GetSessionRestaurantsFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: restaurant/get/
+      Handler: app.lambda_handler
+      Runtime: python3.9
+      Environment:
```
Comment (nabilamin) [inline aws_serverless/template.yaml:90]: We can also get rid of this as well since this specific lambda is not hitting the yelp api

**Review by nabilamin — CHANGES_REQUESTED** [review]
Amazing work!

File: aws_serverless/template.yaml (hunk 1 above)
Comment (JonGeorge) [reply aws_serverless/template.yaml]: Oh shoot! Good catch!

File: aws_serverless/template.yaml (hunk 2 above)
Comment (JonGeorge) [reply aws_serverless/template.yaml]: Makes sense! This has been removed.

**Comment by JonGeorge** [comment]
Ready for a second review!

**Pull request merged**